pyo3-ffi/src/lib.rs

```diff
@@ -0,0 +1,206 @@
+//! Raw FFI declarations for Python's C API.
```

Comment:
This documentation could be fleshed out a bit more. This crate should make sense on its own, so 
- Descriptions of its features
- Descriptions about the various cfg and the pyo3-build-config crate
- Something about the minimum supported Rust and Python versions
- One or two simple code examples using only the C api

You can copy paste most of these from pyo3's lib.rs

Reply:
I have added both documentation and a readme (that for the most part just copies said documentation). Writing these kinds of docs is not my strength so I would welcome some feedback :smile: